prime-router/src/main/kotlin/metadata/LookupTable.kt

```diff
@@ -152,6 +152,20 @@ open class LookupTable : Logging {
         }
     }
 
+    /**
+     * Returns the table as a sequence of map, where each key in the map is the column name. If you have multiple
+     * columns with the same name in a table, the last one wins. So don't do that.
```

Comment:
Nit: you could throw if `colName` is already in `rowMap` if you wanted

Reply:
Ooh I didn't actually right this comment or the corresponding method I'm not sure why it came up that way in the diff.

That's a great point regardless though. I'll make the change and run tests.

Reply:
The behavior comes from an abstraction provided by a dependency and thus is tough to change.